tests/dummy/app/app.js

```diff
@@ -4,7 +4,7 @@ import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
 // ember-browserify requires every package to be imported somewhere in the `app` tree
-import 'npm:common-tags';
+import 'common-tags';
```

Comment:
You should be able to get rid of this line (and the `ember-browserify` dependency in `package.json`) with this change :)